README.md

```diff
@@ -58,4 +58,6 @@ yarn run
 
 This will display an interactive session where you can select the project and command you want.
 
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/grain-lang/grain)
```

Comment:
```suggestion
```

Shall we just delete this so we can get it merged and then decide where to advertise it later?

Reply:
Sure.